modules/google-analytics/classes/class-jetpack-google-amp-analytics.php

```diff
@@ -0,0 +1,130 @@
+<?php
+/**
+ * Add support for Google Analytics e-commerce events for AMP pages.
+ *
+ * @package Jetpack
+ */
+
+/**
+ * Bail if accessed directly
+ */
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Jetpack_Google_AMP_Analytics class.
+ */
+class Jetpack_Google_AMP_Analytics {
+	/**
+	 * Add hooks.
+	 */
+	public function init() {
```

Comment:
Is there a specific reason why you went with this instead of a `__construct()` function?